android/java/org/chromium/chrome/browser/rate/BraveRateDialogFragment.java

```diff
@@ -68,27 +69,48 @@ public void show(@NonNull FragmentManager manager, @Nullable String tag) {
         }
     }
 
-    @SuppressLint("RestrictedApi")
     @Override
-    public void setupDialog(@NonNull Dialog dialog, int style) {
-        super.setupDialog(dialog, style);
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        ((BottomSheetDialog) getDialog())
+                .getBehavior()
+                .setState(BottomSheetBehavior.STATE_EXPANDED);
 
-        final View view = LayoutInflater.from(getContext())
-                                  .inflate(R.layout.brave_rating_dialog_fragment, null);
         clickOnHappyImageView(view);
         clickOnSadImageView(view);
-        dialog.setContentView(view);
     }
 
+    @Nullable
+    @Override
+    public View onCreateView(
+            @NonNull LayoutInflater inflater,
+            @Nullable ViewGroup container,
+            @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.brave_rating_dialog_fragment, container, false);
+    }
+
+    // @SuppressLint("RestrictedApi")
+    // @Override
+    // public void setupDialog(@NonNull Dialog dialog, int style) {
+    //     super.setupDialog(dialog, style);
+
+    //     final View view = LayoutInflater.from(getContext())
+    //                               .inflate(R.layout.brave_rating_dialog_fragment, null);
+    //     clickOnHappyImageView(view);
+    //     clickOnSadImageView(view);
+    //     dialog.setContentView(view);
+    // }
+
```

Comment:
can we remove commented code if not required in future?

Reply:
oops missed it. Good catch